config/config.yaml.dist

```diff
@@ -260,6 +260,9 @@ dex:
 #            user: ""
 #            password: ""
 #            insecure: false
+#            # The path of the directory that contains Banzaicloud default policies
```

Comment:
**Note**: never expected to see Banzai Cloud written this way. 🙂

Reply:
I will fix it :D